app.test.js

```diff
@@ -42,3 +42,59 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: response with 200 status code and returns article given ID", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toMatchObject({
```

Comment:
I would strongly consider using a flexible matcher in this test instead of asserting the article equals an exact copy. Things may change (properties might be added to the article in the future and then this test would fail). Asserting the id is the correct one is definitely still required though